examples/src/main/java/com/mapbox/navigation/examples/core/RouteAlertsActivity.kt

```diff
@@ -71,6 +72,15 @@ class RouteAlertsActivity : AppCompatActivity() {
             *generateLineLayerProperties(Color.DKGRAY)
         )
 
+    private val tunnelNamesTextPropertyId = "tunnel_names_text_property"
+    private val tunnelNamesSource = GeoJsonSource("tunnel_names_source")
+    private val tunnelNamesLayer = SymbolLayer(
+        "tunnel_names_layer",
+        "tunnel_names_source"
+    ).withProperties(
+        *generateTunnelNamesLayerProperties(yellowCircleImageId, tunnelNamesTextPropertyId)
```

Comment:
https://proandroiddev.com/kotlins-vararg-and-spread-operator-4200c07d65e1
https://kotlinlang.org/docs/reference/functions.html#variable-number-of-arguments-varargs

"if we already have an array and want to pass its contents to the function, we use the spread operator (prefix the array with *):" 

the `spread operator`. haven't used it yet, cool 👍